### PR TITLE
DOC-331 PS Guide + DOC-280 Scoping a Migration

### PR DESCRIPTION
**Pull request by EdwardAngert**
**Note:** Please fill out the PR Template to ensure proper processing.

Closes: DOC-280

## Summary

**[Professional Services](https://pantheon.io/docs/professional-services) and [Scoping a Migration](https://pantheon.io/docs/migration-scoping)** - A guide for Pantheon Professional Services dedicates more space to some offerings including Migrations and what to expect when scoping a migration with Professional Services.

## Effect
(Use this section to detail the changes summarized above, or remove if not needed)
The following changes are already committed:

- convert doc to md
- convert PS doc to multipart guide

## Remaining Work
(Remove if not needed)
The following changes still need to be completed:

- [x] style tweak? edit bold text into subheadings?
- [x] copy edit

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6230](https://pr-6230--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `47` | `70` | `23` |
| Best Practice Score | `79` | `79` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-6230` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/25779/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/25779/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6230--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/website-migration-service.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+1. Adding any new functionality that didn't previously exist on the site, which includes:
+
+   - Enabling or disabling S3 or other CDN configurations
+
+   - Switching to Solr from another search platform
+
+1. Any code changes made to the site after the initial migration begins.
+
+   - You may add any code changes to a Multidev and merge them in post-launch, or once you take ownership of the site, move changes in before launch
```
Comment (EdwardAngert) [inline source/content/website-migration-service.md:113]: Trying to find a better way to phrase this, maybe?
> ..., or once you assume ownership of the migrated site, move changes in before launch

**Review by carolyn-shannon — APPROVED** [review]
This looks ok. I think "Known Solutions" seems ambiguous - does it mean identified solutions? Solutions identified as part of the project scope? Wondering if there is there another way to describe this.

**Comment by EdwardAngert** [comment]
@carolyn-shannon I adjusted those lines to try to explain "known solutions" better - once @lauren-kelly has 👍 , I'll move this over to copy review

**Comment by EdwardAngert** [comment]
taking this back so I can include it in the work for DOC-331

**Comment by EdwardAngert** [comment]
once #6329 is merged, we should add `showtoc` to the pages in this guide as well

**Review by carolyn-shannon — APPROVED** [review]
updated, ready for re-review

**Pull request merged**